cypress/integration/Tools/TestUiViewAndLogs.spec.js

```diff
@@ -0,0 +1,84 @@
+/**
+* Copyright (c) Microsoft Corporation. All rights reserved.  
+ * Licensed under the MIT License.
+*/
+
+// -----------------------------------------------------------------------------
+// The Purpose of this Test Spec File is to test the Persistent Logging features
+// and to demonstrate how to skip dependant tests when their predicessor fails.
+// -----------------------------------------------------------------------------
+
+const helpers = require('../../support/Helpers')
+
+// The difference between the 1st and 2nd is where we place the call to SkipRemainingTestsOfSuiteIfFailed()
+describe('1st Test Suite Root', () => {
+  afterEach(helpers.SkipRemainingTestsOfSuiteIfFailed)
+  describe('Level 2-1', () => {
+    it('The FIRST Test', () => {
+      expect(true).to.equal(true)
+    })
+
+    it('The SECOND Test', () => {
+      expect(false).to.equal(true) // Intentionally fail to verify that the next tests will be skipped.
+    })
+
+    it('The THIRD Test', () => {
+      expect(true).to.equal(true)
+    })
+  })
+
+  describe('Level 2-2', () => {
+    it('The FOURTH Test', () => {
+      expect(true).to.equal(true)
+    })
+
+    it('The FIFTH Test', () => {
+      expect(true).to.equal(true)
+    })
+  })
+})
+
+// The difference between the 1st and 2nd is where we place the call to SkipRemainingTestsOfSuiteIfFailed()
+describe('2nd Test Suite Root', () => {
+  describe('Level 2-1', () => {
+    afterEach(helpers.SkipRemainingTestsOfSuiteIfFailed)
```

Comment:
Placing the `Skip...` here causes all remaining tests of the **SUB**suite to be skipped if there is a failure, yet the tests of the level one up will continue to run.